src/client/components/OfferProductsList/OfferProductsList.component.js

```diff
@@ -1,24 +1,45 @@
 import React from 'react';
 import Product from '../SpecialOffers/OfferProductModel.component';
 import './OfferProductsList.styles.css';
+import PropTypes from 'prop-types';
 
-export default function OfferProducts({ products, addToCart, addToFavorites }) {
+export default function OfferProducts({ products, addToCart }) {
```

Comment:
Don't we need addToFavorites here?

Reply:
because it is defined in product component,add to cart is also not required to pass as it is also defined in product component